static/app/components/groupPreviewTooltip/evidencePreview.tsx

```diff
@@ -1,62 +1,41 @@
-import {Fragment, ReactChild, useEffect} from 'react';
+import {ReactChild, useEffect} from 'react';
 import styled from '@emotion/styled';
 
 import KeyValueList from 'sentry/components/events/interfaces/keyValueList';
 import {GroupPreviewHovercard} from 'sentry/components/groupPreviewTooltip/groupPreviewHovercard';
-import {useDelayedLoadingState} from 'sentry/components/groupPreviewTooltip/utils';
+import {
+  useDelayedLoadingState,
+  usePreviewEvent,
+} from 'sentry/components/groupPreviewTooltip/utils';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {EventTransaction} from 'sentry/types';
-import {useApiQuery} from 'sentry/utils/queryClient';
-import useOrganization from 'sentry/utils/useOrganization';
 
 type SpanEvidencePreviewProps = {
   children: ReactChild;
+  groupId: string;
   eventId?: string;
-  groupId?: string;
   projectSlug?: string;
 };
 
 type SpanEvidencePreviewBodyProps = {
-  endpointUrl: string;
+  groupId: string;
   onRequestBegin: () => void;
   onRequestEnd: () => void;
   onUnmount: () => void;
-};
-
-const makeGroupPreviewRequestUrl = ({
-  orgSlug,
-  eventId,
-  groupId,
-  projectSlug,
-}: {
-  orgSlug: string;
   eventId?: string;
-  groupId?: string;
   projectSlug?: string;
-}) => {
-  if (eventId && projectSlug) {
-    return `/projects/${orgSlug}/${projectSlug}/events/${eventId}/`;
-  }
-
-  if (groupId) {
-    return `/issues/${groupId}/events/latest/`;
-  }
-
-  return null;
 };
 
 function SpanEvidencePreviewBody({
-  endpointUrl,
   onRequestBegin,
   onRequestEnd,
   onUnmount,
+  groupId,
+  eventId,
+  projectSlug,
 }: SpanEvidencePreviewBodyProps) {
-  const {data, isLoading, isError} = useApiQuery<EventTransaction>(
-    [endpointUrl, {query: {referrer: 'api.issues.preview-performance'}}],
```

Comment:
we might want to give us the ability to pass query parameters that are not part of the query key

Reply:
Yeah I was thinking the same thing. I went ahead and just removed it here because I don't think this was actually being recorded by anything (as far as I could tell)